invoices/sql_store.go

```diff
@@ -1576,13 +1576,6 @@ func fetchInvoiceData(ctx context.Context, db SQLInvoiceQueries,
 
 	if len(htlcs) > 0 {
 		invoice.Htlcs = htlcs
-		var amountPaid lnwire.MilliSatoshi
-		for _, htlc := range htlcs {
-			if htlc.State == HtlcStateSettled {
```

Comment:
I think we just need to adopt it and also add it if the htlc is in the accepted state, rather then removing it?
https://github.com/lightningnetwork/lnd/blob/1b0f41da48b4dec3db5b51e8b5c28048ecc872b6/invoices/update_invoice.go#L370-L379

Reply:
tested it locally it does work with this change:
```
		for _, htlc := range htlcs {
			if htlc.State == HtlcStateSettled ||
				htlc.State == HtlcStateAccepted {

				amountPaid += htlc.Amt
			}
		}
		invoice.AmtPaid = amountPaid
```

Reply:
Tested it with `go test -v -run=TestMigrationWithChannelDB -tags=kvdb_sqlite` and i think the only correct fix given your db is to use the stored amount paid value. Also discussed this offline.